lib/index.js

```diff
@@ -10,10 +10,20 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-/* globals process, console, Q, JSON, escape, verifyToken, crypto */
+/* globals process, console, SDK_VERSION */
 /*jshint maxcomplexity:15 */
 
-var SDK_VERSION = '<%= sdkversion %>'; // Will be replaced by grunt
+require('./vendor/polyfills');
```

Comment:
I'm also curious about whether we can use a tree-shaker loader with webpack - i.e. one of the reasons we're doing the polyfills is because we didn't want underscore as a dependency, but if we can just pull the functions we want (namely, isArray and indexOf), I think it'd be nice to get away from polyfills altogether (i.e. maybe even use another base64 lib that exports a function rather than polyfills it)